crates/xdp/src/lib.rs

```diff
@@ -40,8 +43,18 @@ pub enum LoadError {
     DefaultPortRangeModified(u16, u16),
 }
 
+pub struct XdpWorker {
```

Comment:
Can we get some documentation on this struct and its fields? Focused on their role rather than what they are.